src/compiler/diagnosticMessages.json

```diff
@@ -2877,10 +2877,6 @@
         "category": "Message",
         "code": 6136
     },
-    "No types specified in 'package.json', so returning 'main' value of '{0}'": {

```

Comment:
This message is not needed any more - now we do the full TS lookup before doing the JS lookup.